1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/misc/NameProtect.java

```diff
@@ -31,16 +34,15 @@ public void onText(final TextEvent event) {
             return;
 
         for (final FriendsConfig.Friend friend : LiquidBounce.fileManager.friendsConfig.getFriends())
-            event.setText(StringUtils.replace(event.getText(), friend.getPlayerName(), ColorUtils.translateAlternateColorCodes(friend.getAlias()) + "§f"));
```

Comment:
StringUtils is faster than `String.replace`

Reply:
Okay, for me it lags as fuck: https://streamable.com/rl8sj3
My friendlist: https://pastebin.com/gUaFhBzm
It is realyl annoying when you tab your friendlist and load it forever.
Do you have a better solution?

Reply:
Try it on your own.